xbmc/cores/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -328,12 +328,27 @@ COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
 {
   CRect src, dst;
   g_renderManager.GetVideoRect(src, dst);
-
-  int width  = MathUtils::round_int(dst.Width());
-  int height = MathUtils::round_int(dst.Height());
-
+  RESOLUTION_INFO res = g_graphicsContext.GetResInfo();
+
+  int frameWidth = res.iWidth;
```

Comment:
g_graphicsContext.GetViewWindow is better but still not ok. this should not bypass renderer. what about vertical shift which seems to be ignored here?
some ssa subs require exact position of video because they can be everywhere on the screen. how do you deal with this requirement? ALIGN_SCREEN feels wrong here.